pkg/snapstore/s3_snapstore.go

```diff
@@ -61,56 +73,57 @@ type S3SnapStore struct {
 	maxParallelChunkUploads uint
 	minChunkSize            int64
 	tempDir                 string
+	sseCreds                sseCredentials
 }
 
 // NewS3SnapStore create new S3SnapStore from shared configuration with specified bucket
 func NewS3SnapStore(config *brtypes.SnapstoreConfig) (*S3SnapStore, error) {
-	credentials, err := getSessionOptions(getEnvPrefixString(config.IsSource))
+	credentials, sseCreds, err := getSessionOptions(getEnvPrefixString(config.IsSource))
```

Comment:
can you please rename `credentials` to `sessionOpts` instead?

Reply:
Done